src/contribute/release-guide.md

```diff
@@ -398,6 +398,40 @@ Staging binaries
 
 Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
+
+### Build python wheels and upload it to test.pypi.org
+
+1. Create your test.pypi.org account in https://test.pypi.org/account/register/ .
+   * Note: Please ping dev@ thread to ask for permissions accessing apache_beam in test.pypi.org.
+1. Fork https://github.com/robertwb/apache-beam-wheels .
```

Comment:
Perhaps this can live in beam/release-tools or something like that?

Reply:
Hey Valentyn, could you please explain more about your ideas?

Reply:
I was thinking that perhaps it would make sense to move necessary tooling to build wheels into some folder in Beam Github repository that contains release tools?

Reply:
From what I know so far, it's better to keep it as a separated repo. How about you @robertwb ?